src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerEndpoint.java

```diff
@@ -78,12 +82,21 @@ public DockerServerEndpoint(String uri, String credentialsId) {
      *
      * <p>
      * Null to indicate whatever Docker picks by default.
+     * @deprecated use #getDockerHost
```

Comment:
Do not deprecate this, it is part of the GUI configuration (getter corresponding to constructor parameter).